docs/getting-started/quick-start.md

```diff
@@ -29,28 +29,189 @@ First, install the necessary package. The command below will install the main CK
 npm install ckeditor5
 ```
 
+Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately. Please note the {@link module:essentials/essentials~Essentials `Essentials`} plugin including all essential editing features.
+
+**Importing and registering UI translations is optional for American English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/ui-language setup section}.
+
+```js
+import { ClassicEditor, Essentials, Bold, Italic, Font, Paragraph } from 'ckeditor5';
+
+import 'ckeditor5/ckeditor5.css';
+
+ClassicEditor
+	.create( document.querySelector( '#editor' ), {
+		plugins: [ Essentials, Bold, Italic, Font, Paragraph ],
+		toolbar: {
+			items: [
+				'undo', 'redo', '|', 'bold', 'italic', '|',
+				'fontSize', 'fontFamily', 'fontColor', 'fontBackgroundColor'
+			]
+		}
+	} )
+	.then( /* ... */ )
+	.catch( /* ... */ );
+```
+
+Pass the imported plugins inside the configuration to the {@link module:editor-classic/classiceditor~ClassicEditor#create `create()`} method and add toolbar items where applicable.
+
+The first argument in the `create()` function is a DOM element for the editor placement, so you need to add it to your HTML page.
+
+```html
+<div id="editor">
+	<p>Hello from CKEditor 5!</p>
+</div>
+```
+
+That is all the code you need to see a bare-bone editor running in your web browser.
+
+## Installing CKEditor&nbsp;5 from CDN
+
+CDN is an alternative method of running CKEditor 5. You can start using it in just a few steps and with a few tags.
+
+Start by attaching a link to style sheets. They contain all styles for the editor's UI and content. You can also include your styles if you like. Refer to the content styles guide for more information.
+
+```html
+<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.css" />
+```
+
+Then, you need to attach the script with the JavaScript code. To simplify imports, you can use the feature available in browsers &ndash; the [import map](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type/importmap). It allows us to map an easy-to-remember specifier (like `ckeditor5`) to the full URL of the file from the CDN. We use this browser feature to share an editor engine code between plugins.
```

Comment:
We need to add infor that the files need to be run with HTTP server otherwise they will fail. @gorzelinski any idea how to phrase it?

Reply:
Good catch. I stumbled upon this error. Maybe something like:

> ES6 modules are subject to the same-origin CORS policy. A local server is required to serve them via HTTP(s) protocol. Directly opening the HTML file will **not** work.

Reply:
I think junior devs won't understand it :smile: 

> To use import maps, you need to run your code on a local server. Just opening the HTML file directly in your browser won’t work because of security rules. These rules (called CORS policy) make sure that your modules are loaded from the same source. So, set up a local server, like \`nginx\`, \`caddy\`, \`http-server\`, to serve your files over HTTP or HTTPS.